src/Oceananigans.jl

```diff
@@ -89,17 +92,17 @@ export
     # Package utilities
     prettytime
 
+# Standard library modules
 using
     Statistics,
     Printf
 
+# Third-party modules
 using
     FFTW,
     JLD,
     NetCDF,
-    StaticArrays
-
-using
+    StaticArrays,
     GPUifyLoops

```

Comment:
will want to switch to `import` so the `Architecture` types don't clash with `GPUifyLoops.Device`?

Reply:
Yes that's true. I'm still using GPUifyLoops v0.1.0 so there's no `GPUifyLoops.Device` yet, but will change this now so moving to v0.2.0 #153 will be easier.